core/frontend/src/main/scala/io/udash/bindings/TextArea.scala

```diff
@@ -40,4 +42,12 @@ object TextArea {
 
     textarea(bind, xs)
   }
+
+  /**
+    * @param property Property to bind.
+    * @param xs Additional Modifiers, don't use modifiers on value, onchange and onkeyup attributes.
+    * @return HTML textarea with bound Property, applied modifiers and nested options.
+    */
+  def apply(property: Property[String], xs: Modifier*): JsDom.TypedTag[html.TextArea] =
+    apply(property, 20 millis, xs:_*)
```

Comment:
Take a look, again. Now `apply(property: Property[String], xs: Modifier*)` is deprecated and works exactly the same as before changes. There is  `debounced` method which applies default debouncing.